### PR TITLE
Add zombified llamas

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Content "Add a zombified version of the llama"

#### Purpose of change
Add a zombie variant of the llama, based off of issue #60362. 

#### Describe the solution
Create the "zllama", a reanimated llama that has lost its wool and can spit acrid cud at the player.

#### Describe alternatives you've considered
Not adding the zllama, or changing it in some way.

#### Testing
All code functions as intended.

#### Additional context
None

### CODE REVIEW

**Comment by zombiemimic** [comment]
zllama doesn't roll off the tongue well, but otherwise I like it

**Comment by descan** [comment]
> zllama doesn't roll off the tongue well, but otherwise I like it


Their wild ancestor is named the vicuña, which puts me in mind of "vicious vicuña." I'm not a huge fan of that, but I also can't think of anything *better*.

**Comment by MNG-cataclysm** [comment]
> Their wild ancestor is named the vicuña, which puts me in mind of "vicious vicuña." I'm not a huge fan of that, but I also can't think of anything _better_.

I like that name better, I've changed it to that. Thank you.

**Comment by Zireael07** [comment]
I like the new name better too.

File: data/json/monsters/zed-animal.json [hunk 1]
```diff
@@ -482,6 +482,48 @@
       "PUSH_VEH"
     ]
   },
+  {
+    "id": "mon_zllama",
+    "type": "MONSTER",
+    "name": { "str_sp": "vicious vicuña" },
```
Comment (Fris0uman) [inline data/json/monsters/zed-animal.json:488]: Should add a comment for translators about the origin of the name

Comment (MNG-cataclysm) [reply data/json/monsters/zed-animal.json:488]: It's done. See line 489.

**Review by Maleclypse — CHANGES_REQUESTED** [review]
Won't this need to either spawn somewhere or have a change to the Llama json adding a line for zombifies into?

**Comment by MNG-cataclysm** [comment]
> Won't this need to either spawn somewhere or have a change to the Llama json adding a line for zombifies into?

I added a zombifies into line for the llamas.

**Comment by MNG-cataclysm** [comment]
> Won't this need to either spawn somewhere or have a change to the Llama json adding a line for zombifies into?

@Maleclypse I added a zombifies into line for the llamas.

**Review by Maleclypse — APPROVED** [review]

**Comment by sadenar** [comment]
Oh before it gets merged, I just realized you have a minor typo in the description where a It's is used where there should simply be an Its.

File: data/json/monsters/zed-animal.json [hunk 2]
```diff
@@ -482,6 +482,49 @@
...
+    "//": "The wild ancestor of the llama is called a vicuña.",
+    "description": "This llama oozes a dark oil from a multitude of lesions and cuts.  It's wool has sloughed off with parts of its skin, revealing pulsating muscles and flesh.  Its mouth drips with a sickly green substance which it seems to be chewing on.",
```
Comment (dseguin) [inline data/json/monsters/zed-animal.json:490]:
```suggestion
    "description": "This llama oozes a dark oil from a multitude of lesions and cuts.  Its wool has sloughed off with parts of its skin, revealing pulsating muscles and flesh.  Its mouth drips with a sickly green substance which it seems to be chewing on.",
```

**Pull request merged**